hips/draw/tests/test_simple.py

```diff
@@ -25,32 +25,32 @@ def get_test_tiles(file_format, survey, order, ipix_list):
-         data_2=2563.6916727348043, data_sum=2992643842.729775, order=3, ipix_list=[450, 451]),
-    dict(file_format='jpg', shape=(2000, 1000, 3), survey='DSS2Red', data_1=[13.040878, 13.040878, 13.040878],
-         data_2=[17.235874, 17.235874, 17.235874], data_sum=155233709.20236143, order=3, ipix_list=[450, 451]),
-    dict(file_format='png', shape=(2000, 1000, 4), survey='AKARI-FIS', data_1=[254., 254., 254., 255.],
-         data_2=[254., 254., 254., 255.], data_sum=586208559.2450126, order=3, ipix_list=[450, 451])
-]
-
-
-@remote_data
-@requires_hips_extra()
-@pytest.mark.parametrize('pars', draw_sky_image_pars)
-def test_draw_sky_image(pars):
-    geometry = make_test_wcs_geometry(case=2)
-    tiles = get_test_tiles(file_format=pars['file_format'], survey=pars['survey'], order=pars['order'], ipix_list=pars['ipix_list'])
-    url = 'https://raw.githubusercontent.com/hipspy/hips-extra/master/datasets/samples/' + pars['survey'] + '/properties'
-    hips_survey = HipsSurveyProperties.fetch(url)
-
-    data = draw_sky_image(geometry=geometry, tiles=tiles, hips_survey=hips_survey, tile_format=pars['file_format'])
-
-    assert data.shape == pars['shape']
-    assert data.dtype == np.float64
-    assert_allclose(np.sum(data), pars['data_sum'])
-    assert_allclose(data[400, 500], pars['data_1'])
-    assert_allclose(data[400, 501], pars['data_2'])
+# draw_sky_image_pars = [
+#     dict(file_format='fits', shape=(2000, 1000), survey='DSS2Red', data_1=2866.0101409848185,
+#          data_2=2563.6916727348043, data_sum=2992643842.729775, order=3, ipix_list=[450, 451]),
+#     dict(file_format='jpg', shape=(2000, 1000, 3), survey='DSS2Red', data_1=[13.040878, 13.040878, 13.040878],
+#          data_2=[17.235874, 17.235874, 17.235874], data_sum=155233709.20236143, order=3, ipix_list=[450, 451]),
+#     dict(file_format='png', shape=(2000, 1000, 4), survey='AKARI-FIS', data_1=[254., 254., 254., 255.],
+#          data_2=[254., 254., 254., 255.], data_sum=586208559.2450126, order=3, ipix_list=[450, 451])
+# ]
+
+
+# @remote_data
+# @requires_hips_extra()
+# @pytest.mark.parametrize('pars', draw_sky_image_pars)
+# def test_draw_sky_image(pars):
+#     geometry = make_test_wcs_geometry(case=2)
+#     tiles = get_test_tiles(file_format=pars['file_format'], survey=pars['survey'], order=pars['order'], ipix_list=pars['ipix_list'])
+#     url = 'https://raw.githubusercontent.com/hipspy/hips-extra/master/datasets/samples/' + pars['survey'] + '/properties'
+#     hips_survey = HipsSurveyProperties.fetch(url)
+#
+#     data = draw_sky_image(geometry=geometry, tiles=tiles, hips_survey=hips_survey, tile_format=pars['file_format'])
+#
+#     assert data.shape == pars['shape']
+#     assert data.dtype == np.float64
+#     assert_allclose(np.sum(data), pars['data_sum'])
+#     assert_allclose(data[400, 500], pars['data_1'])
+#     assert_allclose(data[400, 501], pars['data_2'])
```

Comment:
@cdeil I think this can be updated according to the current code. However, in the current commented-out code the `tiles` are passed in manually, but the updated code computes them after calling `healpix_pixel_indices`.

So, to use this as-it-is I would need to add the missing tiles to `hips-extra` so the test doesn't fail. One other option could be to pass in an optional parameter for `tiles`, so only the provided tiles will be draw. Do you have any opinion on this?